asciidoc/courses/importing-data/modules/4-importing-data-cypher/lessons/2-c-importing-with-cypher/lesson.adoc

```diff
@@ -12,8 +12,8 @@ This challenge has 7 steps:
 . Ensure that all constraints exist in the graph.
 . Import _Movie_  and _Genre_ data.
 . Import _Person_ data.
-. Import  the ACTED_IN relationships.
-. Import  the DIRECTED relationships.
+. Import the ACTED_IN relationships.
+. Import the DIRECTED relationships.
```

Comment:
For database elements, it may be worth spending some time to standardise how we display them - italics vs monospace.  This is fine for now though.